bin/crystal

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
I thought `/bin/sh` was actually required by POSIX or something like that.

Is there any system in which you don't have `/bin/sh` available, but you have `/usr/bin/env` and `sh` in some other path?

I'm totally for rolling back to `/usr/bin/env sh` if it is true that `/usr/bin/env` is _more standard_ than `/bin/sh` - but I wouldn't think that's true.